hibernate-search-elasticsearch-quickstart/src/test/java/org/acme/hibernate/search/elasticsearch/service/LibraryResourceInGraalIT.java

```diff
@@ -15,6 +15,21 @@
  */
 package org.acme.hibernate.search.elasticsearch.service;
 
+/*
```

Comment:
That looks like too many license headers...

Reply:
yeah, nice catch :)

Reply:
PR updated